dash/dash.py

```diff
@@ -378,15 +378,27 @@ def _relative_url_path(relative_package_path='', namespace=''):
 
         srcs = []
         for resource in resources:
+            is_dynamic_resource = (
+                'dynamic' in resource and
+                resource['dynamic'] is True
+            )
```

Comment:
Not sure what the preferred indentation pattern is for Python -- went with something that seemed natural with the syntax

Reply:
No `()` and single line, we prefer explicit:  `if 'dynamic' in resources and resource.get('dynamic')`. Also `.get` over `[key]` to avoid unnecessary KeyErrors. Only use `is` for comparing pointer references, ie `is None` to assert null and not just falsy.

Reply:
Did this because the line is too long for flake8's taste.

Reply:
How do I one line this and make flake8 happy? Apart from an actual function, etc.

Reply:
Break them using `\` at the end of the line.

Reply:
With '== True' I get this from flak8
dash/dash.py:382:41: E712 comparison to True should be 'if cond is True:' or 'if cond:'

Reply:
Getting the same error from pylint

Reply:
And now this.. dash/dash.py:355:4: R0912: Too many branches (13/12) (too-many-branches

I guess the additional `if` triggered this rule.

Reply:
I prefer `if cond` to `is True`, `is True` is when you need to assert that it the actual `True` and not another type that evaluate to True in a condition like a string.

Reply:
Ah! Misread! No need for the explicit comparison...

Reply:
Add `# pylint: disable=too-many-branches` to the start of the function.